openwisp_users/multitenancy.py

```diff
@@ -2,6 +2,30 @@
 from django.db.models import Q
 from django.utils.translation import ugettext_lazy as _
 
+from .models import OrganizationUser, User
+
+
+def multitenant_behaviour_for_user_admin(self, request):
```

Comment:
seems this function is only used in `MultitenantAdminMixin` right?
In this case I think it makes more sense to have this as a method of `MultitenantAdminMixin` instead of a stand-alone function. Can you do this change please?

Reply:
I think I actually wanted to use it as a static function somewhere, using the decorator, I don't remember now. Anyway, I just changed it.